kratos/utilities/coordinate_transformation_utilities.h

```diff
@@ -753,7 +672,7 @@ class CoordinateTransformationUtils {
 	///@name Protected Operations
 	///@{
 
-	template<unsigned int TDim, unsigned int TBlockSize, unsigned int TSkip = 0>
+	template<unsigned int TDim, unsigned int TBlockSize, unsigned int TSkip = 0, bool TRevertRotation = false>
```

Comment:
This is no longer needed right?

Reply:
Unfortunately, the rotate operation on the LHS in `MPMApplication` is implemented differently from the core. We thus require the two versions of `RotateAux` with the ability to revert the rotation applied. Since this is already an auxillary function, making the changes here directly makes more sense in my opinion (as compared to extracting the contents to yet another aux function). I can add some documentation here if you think the changes might be confusing.

Reply:
Makes sense for me. I just wanted to confirm that this was not a leftover.